code/modules/mob/mob_say.dm

```diff
@@ -21,7 +21,7 @@
 
 ///whisper a message
 /mob/proc/whisper(message, datum/language/language=null)
-	say(message, language) //only living mobs actually whisper, everything else just talks
+	SSspeech_controller.queue_say_for_mob(src, message)
```

Comment:
Where does the language go? Also why do this here?

Reply:
oh oops i missed that

Reply:
now adds support for additional arguments